api/handler/acl.go

```diff
@@ -321,7 +321,7 @@ func (h *handler) PutObjectACLHandler(w http.ResponseWriter, r *http.Request) {
 		VersionID: versionID,
 	}
 
-	objInfo, err := h.obj.GetObjectInfo(r.Context(), p)
+	extInfo, err := h.obj.GetObjectInfo(r.Context(), p)
```

Comment:
I think it is not obvious enough that `extInfo` means extended info. Maybe it'd be better to name it as `extendedInfo` or `extendedObjectInfo`?